### ISSUE TITLE
Problem installing @types with yarn v1.18.1

### ISSUE THREAD

**Issue by danrevah** [yarnpkg/yarn#2322]
I'm getting an error while trying to install @types scoping with Yarn.

```bash
$ yarn
yarn install v0.18.1
info No lockfile found.
[1/4] 🔍  Resolving packages...
error Couldn't find package "@types/jasmine" on the "npm" registry.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
````

I managed to have a temporal fix while adding to `~/.yarnrc`:

```
"@types:registry" "https://registry.npmjs.org/"
```

Shouldn't yarn have a fallback to the registry "https://registry.npmjs.org" in case of scoping  ?

**Comment by BendingBender** [comment]
Yes, this fixes the problem for me, too. Thank you very much. What I don't understand is that it fails on one of my machines but not on the other with the exactly same `package.json`, `yarn.lock` and yarn version. That's a mystery to me.

**Comment by danrevah** [comment]
Any updates about this?

Angular 2 Pipes: https://github.com/danrevah/ngx-pipes
Regards,

Dan

**Comment by randak** [comment]
Is this issue resolved now? Seems to be fixed here: https://github.com/yarnpkg/yarn/issues/825 and is working for me locally.
